packages/client/src/views/VerifyCertification/UnprotectedPage.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { ReactNode } from 'react'
+import styled from 'styled-components'
+import { Spinner } from '@opencrvs/components'
+import { useDispatch } from 'react-redux'
+import { configAnonymousUserLoaded } from '@client/offline/actions'
+import { IApplicationConfig, referenceApi } from '@client/utils/referenceApi'
+
+const StyledSpinner = styled(Spinner)`
+  position: absolute;
+  margin-left: -24px;
+  margin-top: -24px;
+  top: calc(50% - 20px);
+  left: 50%;
+  width: 40px;
+  height: 40px;
+`
+
+const UnprotectedPage = (props: { children: ReactNode }) => {
```

Comment:
This components seems to be defined twice. See packages/client/src/components/UnprotectedRoute.tsx